otx/core/data/adapter/detection_dataset_adapter.py

```diff
@@ -40,8 +41,10 @@ def get_otx_dataset(self) -> DatasetEntity:
                             self.task_type in (TaskType.INSTANCE_SEGMENTATION, TaskType.ROTATED_DETECTION)
                             and ann.type == AnnotationType.polygon
                         ):
-                            if self._is_normal_polygon(ann):
+                            if isinstance(ann, Polygon):
```

Comment:
I'm not sure why `ann.type == AnnotationType.polygon` couldn't filter out `Bbox`.

If you want to use `isinstance(ann, Polygon)`, then edit above `if` sentence like below.
```python3
if (
    self.task_type in (TaskType.INSTANCE_SEGMENTATION, TaskType.ROTATED_DETECTION) and isinstance(ann, Polygon)
):
```

Reply:
yeah, you're right. I reverted the change.